app/views/create/fromimage.html

```diff
@@ -113,6 +117,10 @@
                               </div>
                               <div class="help-block">Optional branch, tag, or commit.</div>
                             </div>
+                          </div>
+                          </div>
+
+                          <div click-to-reveal ng-click="advancedOptions = !advancedOptions" link-text="Show advanced routing, build, and deployment options">
```

Comment:
If we're also showing things outside the `click-to-reveal` directive, I wouldn't use the directive. Just have a link that toggles `advancedOptions` and have an `ng-if="advancedOptions"` here.